### PR TITLE
Ops

### PR DESCRIPTION
**Pull request by charlieboles**
Operationalize experience, not ready for a merge, created as a request form @iansk 

Most of this is new functionality that is limited to the ops experience, however, there were some slight tweaks to the header js and CSS to detect for touch devices.

Test URLs:
- Before: https://main--prisma-cloud-docs-website--hlxsites.hlx.page/en/operationalize
- After: https://ops--prisma-cloud-docs-website--hlxsites.hlx.page/en/operationalize
- After 2: https://ops--prisma-cloud-docs-website--hlxsites.hlx.page/en/operationalize#secure-the-source
- After 3: https://ops--prisma-cloud-docs-website--hlxsites.hlx.page/en/operationalize#secure-the-infrastructure
- After 4: https://ops--prisma-cloud-docs-website--hlxsites.hlx.page/en/operationalize#secure-the-runtime

### CODE REVIEW

**Comment by iansk** [comment]
@charlieboles We want @maxakuru to review this PR. Can you please provide a summary of the changes you made in common code/blocks that effect other parts of the site. This is where Max will focus his review

@maxakuru Here's the experience Charlie built for us:

https://ops--prisma-cloud-docs-website--hlxsites.hlx.page/en/operationalize#secure-the-source

File: blocks/intro/intro.css [hunk 1]
```diff
@@ -1,50 +1,104 @@
...
+
+.header-section > p:first-child {
+  display: flex;
+  gap: var(--spacing--5);
+  margin: 0 auto;
+  text-align: center;
+}
+
+.header-section > p:first-child:before {
+  background: url('/prisma/prisma-cloud/assets/cloud-logo.png') 0 0 no-repeat;
```
Comment (maxakuru) [inline blocks/intro/intro.css:38]:
```suggestion
  background: url('/assets/cloud-logo.png') 0 0 no-repeat;
```

File: icons/Cloud Logo 2 (2).svg
Comment (maxakuru) [inline icons/Cloud Logo 2 (2).svg:1]: I don't see where this is used? and any reason for the `2 (2)`?

File: scripts/scripts.js [hunk 2]
```diff
@@ -1031,3 +1031,202 @@ export function lerp(start, end, amt) {
...
+  element.classList.add(targetClass);
+
+  setTimeout(() => {
+    element.style.opacity = 1;
+    element.classList.add('stagger-transitions');
+  }, 100);
+};
+
+// Lottie animations for each categoiry
+const LOTTIE_PATHS = {
```
Comment (maxakuru) [inline scripts/scripts.js:1137]: these seem specific to the intro block, maybe we could either 
1. make `showRoute()` generic and pass the lottie path (or path map like this object) as a param
or
2. move the functions that are specific to the intro block into that block's js
wdyt?

**Review by maxakuru — CHANGES_REQUESTED** [review]
Looks like there's a bug in this where you're requesting resources multiple times, performance is pretty bad because of this. I also think lottie.min.js can be moved lower in the request chain. Couple other minor comments

Pic related:

Screenshot 2023-10-12 at 12 14 27 PM

File: blocks/article/article.js [hunk 3]
```diff
@@ -301,7 +301,7 @@ const decorateTitles = (block) => {
     articleTitle.setAttribute('data-docs-heading', true);
 
     articleTitle.innerHTML = `
-          <div class="anchor"></div>
+          <div class="anchor" style="height: 50px"></div>
```
[the remark this answers is not in the document]
Comment (charlieboles) [reply blocks/article/article.js]: Sure

File: blocks/intro/intro.css (hunk 1 above)
Comment (charlieboles) [reply blocks/intro/intro.css]: Updated

File: icons/Cloud Logo 2 (2).svg
Comment (charlieboles) [reply icons/Cloud Logo 2 (2).svg:1]: Removed

File: scripts/scripts.js (hunk 2 above)
Comment (charlieboles) [reply scripts/scripts.js]: So I can just scope it to inside the function, intro and category both make use of it.

**Pull request merged**